### PR TITLE
Change tracking logs S3 check

### PR DESCRIPTION
**Pull request by markbreedlove**
Get rid of the malfunctioning Salt scheduled task and replace it with a shell script on cron.

Is this halfway sane?

We can add the xPro tracking logs bucket when we get that functioning for xPro again.

### CODE REVIEW

File: salt/master_utils/install_tracking_s3_monitor.sls [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        #!/usr/bin/env bash
+        BUCKETS="odl-residential-tracking-data"
+        # -e means fail the script on error ...
+        set -e
+        today=`date +"%F"`
+        for bucket in $BUCKETS; do
+            # ... and this will return error status for no matching objects:
+            aws s3 ls s3://$bucket/logs/$today
+        done
+        curl -s '{{ __vault__::secret-operations/global/healthchecks/mitx-tracking-s3>data>value }}'
```
Comment (blarghmatey) [inline salt/master_utils/install_tracking_s3_monitor.sls:15]: This needs to be passed as pillar data. The vault pillar renderer doesn't work in state files.

Comment (markbreedlove) [reply salt/master_utils/install_tracking_s3_monitor.sls:15]: Would the `master.config` pillar be the best place? Or one of the neighboring ones under `master`?

Comment (markbreedlove) [reply salt/master_utils/install_tracking_s3_monitor.sls]: I've made an attempt to fix it in commit 98a25d34
Can change it, if necessary!

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**